noinst/specs/update_specs.R

```diff
@@ -39,10 +44,12 @@ SPECS <- as_tibble(SPECS)
 
 tbl_txt <-
   SPECS |>
-  filter(is.series == TRUE) |>
-  filter(is.save == TRUE) |>
-  select(spec, long, short) |>
-  transmute(new = paste("#'", spec, "\\tab", long, "\\tab", short, "\\cr")) |>
+  filter(
+    is.series == TRUE,
+    is.save == TRUE
+  ) |>
+  select(spec, long, short, description) |>
+  transmute(new = paste("#'", spec, "\\tab", long, "\\tab", short, "\\tab", description, "\\cr")) |>
```

Comment:
Main issue: we need something to ASCIIfy `description`. A ASCII representation of UTF-8 (like `"fa\u00E7ile"` should be fine and I am sure there are convenient ways how this could be done.

Reply:
You may also have to update the `WORDLIST`, using `spelling::update_wordlist()`.

Reply:
Hmm, interesting. Looks like some descriptions also contain formula-like expressions that may need special treatment.

Reply:
See [5296a81](https://github.com/christophsax/seasonal/pull/301/commits/5296a81b2b594450a0f1c2098cbcd709aea1db7b)

Reply:
But the ASCII problem remains. Have you seen the `R CMD check`?

Reply:
Hmm, it works for me and the GHA-Checks also pass.